### PR TITLE
Fixed phpdoc

### PR DESCRIPTION
**Pull request by core23**

### CODE REVIEW

**Comment by OskarStark** [comment]
Status: Reviewed

ping @Soullivaneuh

File: Model/Category.php [hunk 1]
```diff
@@ -62,6 +97,8 @@ public function getName()
     public function setEnabled($enabled)
     {
         $this->enabled = $enabled;
+
+        return $this;
```
Comment (Soullivaneuh) [inline Model/Category.php]: AFAIK, nothing says that self should be returned on the interface.

BTW, this is not related to the PR (fix php doc). If you want to improve it, submit it on another one.

Comment (OskarStark) [inline Model/Category.php]: :+1:  for separating this in 2 PR's

**Comment by core23** [comment]
Removed ``return $this;`` from models. After merging this, I'll create a seperate PR for the return stuff.

**Comment by OskarStark** [comment]
RTM IMO

ping @Soullivaneuh

**Comment by Soullivaneuh** [comment]
OK for me. Let's merge once Travis will be green.

**Comment by core23** [comment]
Travis is green :)

**Comment by OskarStark** [comment]
Thank you @core23
